conans/client/graph/profile_node_definer.py

```diff
@@ -28,6 +28,7 @@ def initialize_conanfile_profile(conanfile, profile_build, profile_host, base_co
         _initialize_conanfile(conanfile, profile_build, settings_build.copy(), ref)
     else:
         _initialize_conanfile(conanfile, profile_host, settings_host, ref)
+    conanfile.profile_build = profile_build
```

Comment:
Not sure about this, lets discuss, we might want ``conf_build`` and ``options_build`` instead.

Reply:
Let's discuss it